pkg/sessionctx/variable/sysvar.go

```diff
@@ -793,6 +793,12 @@ var defaultSysVars = []*SysVar{
 		vardef.SetMaxDeltaSchemaCount(TidbOptInt64(val, vardef.DefTiDBMaxDeltaSchemaCount))
 		return nil
 	}},
+	{Scope: vardef.ScopeGlobal, Name: vardef.TiDBEnablePointGetCache, Value: BoolToOnOff(vardef.DefTiDBPointGetCache), Hidden: true, Type: vardef.TypeBool, SetGlobal: func(_ context.Context, s *SessionVars, val string) error {
+		s.EnablePointGetCache = TiDBOptOn(val)
+		return nil
+	}, GetGlobal: func(_ context.Context, s *SessionVars) (string, error) {
```

Comment:
`GetGlobal` here is is unnecessary, it returns the value of the session instead of the value of the real global